pootle/templates/browser/_actions.html

```diff
@@ -1,5 +1,6 @@
 {% load i18n %}
 <ul>
+  {% if not can_suggest_only %}
```

Comment:
It doesn't help. I still see the "Fix critical errors" and "Continue translation" links.